pkg/parser/digester.go

```diff
@@ -230,10 +230,11 @@ func (d *sqlDigester) normalize(sql string, keepHint bool, forBinding bool) {
 		}
 
 		d.reduceLit(&currTok)
-
-		// Apply binding matching specific rules
-		if forBinding {
-			// IN (?) => IN ( ... ) #44298
+		if isPlanReplayer {
+			// Apply for plan replayer matching specific rules
```

Comment:
pls make the comment more detailed, why for this code special for plan replayed reload binding and so on